go/libraries/doltcore/env/environment.go

```diff
@@ -56,6 +58,20 @@ const (
 	tempTablesDir = "temptf"
 )
 
+func initializeCommitHooks(ctx context.Context, dEnv *DoltEnv) (postCommitHooks []datas.CommitHook, err error) {
+	backupUrl := os.Getenv(doltdb.ReplicateToRemoteKey)
+	if backupUrl != "" {
+		var destDB *doltdb.DoltDB
+		_, destDB, err = CreateRemote(ctx, "backup", backupUrl, nil, dEnv)
```

Comment:
This isn't the right interface here. Let the user name the remote, use another env var (or put it in config as a followup)

Reply:
leaning towards having a user indicate a named repo-state remote, and passing `ReplicateToRemoteKey = <remote-name>` as the env variable

Reply:
if you think that's a bad idea and backups should be namespace separated lmk before standup